its/plugin/projects/duplications/go/duplication/another.go

```diff
@@ -0,0 +1,41 @@
+package samples
```

Comment:
In the `its/plugin/projects/duplications` folder we can only keep the `scala` folder and delete `ruby` and `go`